source/MissionAction.cpp

```diff
@@ -33,7 +33,15 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 using namespace std;
 
 namespace {
-	void DoGift(PlayerInfo &player, const Outfit *outfit, int count, UI *ui)
+	void DoGiftShip(PlayerInfo &player, const Ship *modelName, const std::string &name, UI *ui)
```

Comment:
The const Ship pointer points to a complete model, not just to a modelname, so I suggest to call the variable model.

Reply:
```suggestion
	void DoGiftShip(PlayerInfo &player, const Ship *model, const std::string &name, UI *ui)
```
Ok, will change it then.